### PR TITLE
Fix empty state text for Application's Advanced configuration table #20262

### PR DESCRIPTION
**Pull request by veniceofcode**
https://github.com/stolostron/backlog/issues/20262

Signed-off-by: Raghuram Banda <rbanda@redhat.com>

### CODE REVIEW

File: frontend/src/routes/Applications/components/ToggleSelector.tsx [hunk 1]
```diff
@@ -1,7 +1,8 @@
+/* eslint-disable no-debugger */
```
Comment (chenz4027) [inline frontend/src/routes/Applications/components/ToggleSelector.tsx:1]: We would need to remove this line

**Review by chenz4027 — CHANGES_REQUESTED** [review]

File: frontend/src/routes/Applications/components/ToggleSelector.tsx (hunk 1 above)
Comment (veniceofcode) [reply frontend/src/routes/Applications/components/ToggleSelector.tsx]: updated and pushed the change

**Pull request merged**